### ISSUE TITLE
Bug on Long Pages

### ISSUE THREAD

**Issue by halfempty** [everythingtype/calcott#47]
@ncalcott I have this note:

> Infinite scroll on long pages (like Black & White tag page)

Is the issue that on pages with lots of images the grid is misfiring, resulting in some images being the wrong size and some overlapping?

**Comment by ncalcott** [comment]
No, it's that some of the larger index categories, like http://www.nicholascalcott.com/index/portrait/ seem to go on forever...

**Comment by ncalcott** [comment]
But yes, at the moment there does seem to be this bug.  That only began recently, though - it wasn't behavior I was encountering prior to this round of development...
